src/blocks/Banner/Banner.tsx

```diff
@@ -48,14 +50,18 @@ export const Banner: React.FC<BannerProps> = ({
                 [PaddingsDirections.top]: paddingTop,
                 [PaddingsDirections.bottom]: paddingBottom,
             }}
+            qa={qas.wrapper}
```

Comment:
destructurization for qas (or commonQA)

Reply:
If you do not mind, I leave it as it is. Because `qaAttributes.wrapper` has more context than just `wrapper`